client/src/assets/scss/foundation/_settings.scss

```diff
@@ -886,7 +881,7 @@ $grid-container: $global-width;
 $grid-columns: 12;
 $grid-margin-gutters: (
   small: 20px,
-  medium: 30px
+  medium: 30px,
 );
 $grid-padding-gutters: $grid-margin-gutters;
 $grid-container-padding: $grid-padding-gutters;
```

Comment:
ill allow these edits since most are innocuous, but we should rarely edit the default foundation settings, and instead override them with our own classes. these changes are okay though